doc/index.rst

```diff
@@ -19,14 +19,38 @@ across computing environments such as `Docker <https://docker.com>`__, `Conda
     :doc:`docs:tutorials/running-a-workflow` tutorial first and then come back
     here.
 
+
```

Comment:
I'm not sure a TOC is necessary on the splash page, but if so I'd recommend the "Big Picture" section to be lifted above the TOC as it's getting buried at the moment.

Reply:
Yeah. When I was working on these changes, I thought about moving the "Big Picture" diagram up because it's definitely buried now. But I realized it's a bit out of date, so I didn't want to make it more prominent yet. The doc home page here could definitely use some more improvements such as that and more consideration of what's presented (e.g. TOC or no TOC). Agreed that a full TOC may not be necessary, but also the current "Commands" TOC on the home page is used as the [target of a redirect](https://github.com/nextstrain/cli/blob/75afe1227449603db8e4a0799a227c88be5e990d/doc/redirects.yaml#L11-L17) for the old `doc/commands/index.rst` page. Anyway, I wanted to postpone a full home page rework for now.